fhirclient/models/account.py

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
-#  Generated from FHIR 4.0.0-a53ec6ee1b (http://hl7.org/fhir/StructureDefinition/Account) on 2019-05-07.
-#  2019, SMART Health IT.
+#  Generated from FHIR 4.0.1-9346c8cc45 (http://hl7.org/fhir/StructureDefinition/Account) on 2024-07-15.
+#  2024, SMART Health IT.
```

Comment:
OK so all the files in `fhirclient/models/` are "owned" by fhir-parser. And I changed the templates (see above) to make sure that all the actual models themselves would only see real 4.0.0 -> 4.0.1 changes and not artifacts of changes in the generator.

I've gone through all the models myself and confirmed that you only see:
- Comment changes like this
- Comment changes on some properties (usually enums) that list more properties than previously.

The test files however, I didn't find as easy a way to avoid churn in. But it was less important, so I didn't try very hard. Sorry. But hopefully those don't need close review.

We're still not running the tests during any CI - I hope to fix that later, but wanted to get on a known-good model-generation baseline first. The old generated tests were wonky.